### PR TITLE
Update Bbo experiments

### PR DESCRIPTION
**Pull request by teytaud**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](https://facebookresearch.github.io/nevergrad/contributing.html) document and completed the CLA (see [**CLA**](https://facebookresearch.github.io/nevergrad/contributing.html#contributor-license-agreement-cla)).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: nevergrad/benchmark/experiments.py [hunk 1]
```diff
@@ -86,6 +87,22 @@ def keras_tuning(
...
+        "NGOpt10",
+        "NGOpt12",
+        "NGOpt13",
+        "NGOpt14",
+        "NGOpt15",
+        "NGOpt16",
+        "NGOpt21",
+        "NGOpt36",
+        "NGOpt38",
+        "NGOpt",
```
Comment (jrapin) [inline nevergrad/benchmark/experiments.py:104]: please use an optimizer group instead, that will avoid duplication

Comment (teytaud) [reply nevergrad/benchmark/experiments.py]: Done.

File: nevergrad/benchmark/experiments.py [hunk 2]
```diff
@@ -115,6 +117,22 @@ def mltuning(
...
+    ]
```
Comment (jrapin) [inline nevergrad/benchmark/experiments.py:135]: use an optimizerlist, why not "ngopts" directly? you can also have a reduced ngopts

File: nevergrad/benchmark/optgroups.py [hunk 3]
```diff
@@ -41,6 +41,11 @@ def get_optimizers(*names: str, seed: tp.Optional[int] = None) -> tp.List[Optim]
     return optims
 
 
+@registry.register
+def ngopts() -> tp.Sequence[Optim]:
+    return [o for o in optimizerlib_registry if "NGOpt" in o] + ["Shiwa"]
```
Comment (jrapin) [inline nevergrad/benchmark/optgroups.py:46]: Better check for all subclasses of NGOptBase
something like `[name for name, cls in optimizerlib_registry.items() if issubclass(cls, NGOptBase)]`
Relying on name is not robust on the long term

**Review by jrapin — CHANGES_REQUESTED** [review]

File: nevergrad/benchmark/experiments.py (hunk 1 above)
Comment (jrapin) [reply nevergrad/benchmark/experiments.py]: only in 1 place out of 2 it seems

**Comment by jrapin** [comment]
no idea what the pytest error is, probably needs rebasing master

**Pull request closed without merging**